app/src/main/java/org/fossasia/openevent/general/welcome/WelcomeFragment.kt

```diff
@@ -91,8 +99,8 @@ class WelcomeFragment : Fragment() {
     }
 
     private fun redirectToAuth() {
-        Navigation.findNavController(rootView).navigate(WelcomeFragmentDirections.actionWelcomeToAuth(
-                redirectedFrom = WELCOME_FRAGMENT, showSkipButton = true)
-        )
+        Navigation.findNavController(rootView)
+            .navigate(WelcomeFragmentDirections
+                .actionWelcomeToAuth(redirectedFrom = WELCOME_FRAGMENT, showSkipButton = true))
```

Comment:
This has to change because of the build failure the check was not able to pass because the line length was greater than 120 which is one of the checks in ktlint